cmd/kube-proxy/app/server.go

```diff
@@ -704,3 +701,16 @@ func getConntrackMax(config kubeproxyconfig.KubeProxyConntrackConfiguration) (in
 	}
 	return 0, nil
 }
+
+// CleanUpAndExit remove iptables rules and exit if success return nil
+func (s *ProxyServer) CleanUpAndExit() error {
+	// remove iptables rules and exit
```

Comment:
nit: this comment can be removed since we have the same comment for the method now.